lib/src/header_parser/parser.dart

```diff
@@ -60,7 +55,12 @@ void initParser(Config c) {
       strings.ffigenFolderName,
       strings.dylibFileName,
     );
-    clang = clang_types.Clang(DynamicLibrary.open(fullDylibPath));
+
+    // Initialise global variables.
+    initialiseGlobals(
```

Comment:
Please check whether you use UK (initialise) or US (initialize) spelling consistently in the repo.

Reply:
Updated it to US spelling. (also used in sqlite, libclang)